server.go

```diff
@@ -5,13 +5,15 @@
 package main
 
 import (
+	"context"
 	"fmt"
 	"net"
 	"net/http"
 	"os"
 	"strings"
+	"sync"
 
-	"github.com/gorilla/context"
+	gorillactx "github.com/gorilla/context"
```

Comment:
I'm not sure we should be using `gorilla/context`. [The repo](https://github.com/gorilla/context) is read-only and seems to be deprecated in favour of stdlib `context.Context`

Reply:
I can't find where this is doing anything, so I removed it.

In the entire code base it is only used here https://github.com/decred/dcrstakepool/blob/85435210c6a561f541e39e9c629ac2b67906764f/server.go#L183

It looks like the context stores its own data maps, but they will contain no data to clear unless it is previously set https://github.com/gorilla/context/blob/master/context.go

We are using gorilla csrf, but it says nothing about calling clear in its example https://github.com/gorilla/csrf/blob/7b29b05df5ae7dc777b92fab3dadee1b15d15ed6/csrf.go#L78-L126

Reply:
And then I found this https://github.com/gorilla/sessions/issues/121
so I dunno.  I'll create an issue about it, since I'm not sure if this needed, or a problem.